ethereum/permissioning/src/main/java/tech/pegasys/pantheon/ethereum/permissioning/node/NodePermissioningController.java

```diff
@@ -65,8 +65,7 @@ public void startPeerDiscoveryCallback(final Runnable peerDiscoveryCallback) {
     return syncStatusNodePermissioningProvider;
   }
 
-  @VisibleForTesting
-  List<NodePermissioningProvider> getProviders() {
+  public List<NodePermissioningProvider> getProviders() {
```

Comment:
Given that we are exposing our list of providers, would it make sense to change it to an unmodifiable list?

Reply:
Will go private again in PAN-2449